js/models/conversations.js

```diff
@@ -167,7 +167,7 @@
 
       if (this.id === this.ourNumber) {
         this.set({ friendRequestStatus: FriendRequestStatusEnum.friends });
-      } else if (lokiP2pAPI) {
+      } else if (typeof lokiP2pAPI !== 'undefined') {
```

Comment:
What does this change do?

Reply:
handles js errors during registration (lokiP2pAPI isn't defined in registration)